nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -93,10 +95,11 @@ class BtActionNode : public BT::ActionNodeBase
 
     // Make sure the server is actually there before continuing
     RCLCPP_DEBUG(node_->get_logger(), "Waiting for \"%s\" action server", action_name.c_str());
-    if (!action_client_->wait_for_action_server(1s)) {
+    if (!action_client_->wait_for_action_server(wait_for_service_timeout_)) {
       RCLCPP_ERROR(
-        node_->get_logger(), "\"%s\" action server not available after waiting for 1 s",
-        action_name.c_str());
+        node_->get_logger(), "\"%s\" action server not available after waiting for %f s",
```

Comment:
```suggestion
        node_->get_logger(), "\"%s\" action server not available after waiting for %0.2f s",
```

No reason to be crazy high accuracy for this

Reply:
You are totally right, but this is in the main branch as well. Should I create a PR targeting the main branch fixing this, which will be backported later? Or do you want me to include it here as well?
https://github.com/ros-navigation/navigation2/blob/7737a2fafa27c18efef4689582799bdce6aeb1df/nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp#L103

Reply:
Do it here as well please